### PR TITLE
etcd-tester: continue for deadline exceeded

### PR DESCRIPTION
**Pull request by gyuho**
For https://github.com/coreos/etcd/issues/4477. We confirmed that PUT requests were timing out
and returning when it triggered at the same moment as leader failure. This saves those requests by continuing in the for loop.

/cc @xiang90

### CODE REVIEW

File: tools/functional-tester/etcd-tester/stresser.go [hunk 1]
```diff
@@ -93,6 +93,9 @@ func (s *stresser) Stress() error {
 				})
 				putcancel()
 				if err != nil {
+					if grpc.ErrorDesc(err) == context.DeadlineExceeded.Error() {
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/stresser.go]: let's add comment for this line.

Comment (gyuho) [inline tools/functional-tester/etcd-tester/stresser.go]: Just added comments. PTAL. Thanks.

File: tools/functional-tester/etcd-tester/stresser.go [hunk 2]
```diff
@@ -93,6 +93,15 @@ func (s *stresser) Stress() error {
...
+						// This retries when request is triggered at the same time as
+						// leader failure. When we terminate the leader, the request to
+						// that leader cannot be processed, and times out. Also requests
+						// to followers cannot be forwarded to the old leader so timing out
+						// as well. We want to keep stressing until the cluster elects a
+						// new leader to process requests.
+						continue
```
Comment (gyuho) [inline tools/functional-tester/etcd-tester/stresser.go]: Made the comment more detailed.

**Comment by xiang90** [comment]
LGTM

**Comment by gyuho** [comment]
Thanks. Will redeploy testing cluster with this patch plus boltdb patch.

**Pull request merged**